spring/src/main/java/com/connexta/transformation/rest/spring/TransformApiTransform.java

```diff
@@ -50,7 +54,13 @@
         @ApiResponse(
             code = 201,
             message =
-                "The transformation request was accepted for processing. The URI for polling the status is returned in the Location header of the response. "),
+                "The transformation request was accepted for processing. The URI for polling the status is returned in the Location header of the response. ",
```

Comment:
```suggestion
                "The transformation request was created for processing. The URI for polling the status is returned in the Location header of the response. ",
```

Reply:
From the client perspective accepted makes more sense. They are giving us the request to process. They don't need to know we are creating our own request internally. Right?

Reply:
Well it's because we decided to use status code `201 Created` for this case instead of `202 Accepted`, so I think he's pointing out that we should use the same verbiage in the response message. `202 Accepted` "indicates that the request has been received but not yet acted upon." Source: [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)